### PR TITLE
Fix anet video downloader

### PR DESCRIPTION
**Pull request by dreamerlin**
This PR updates ActivityNet video download steps since the original one is used the annotation from BSN repo rather than the official one. The number of items in BSN annotation is less than that in official.

### CODE REVIEW

File: tools/data/activitynet/README.md [hunk 1]
```diff
@@ -40,24 +40,42 @@ bash download_features.sh
...
 Then, you can run the following script to prepare videos.
 The codes are adapted from the [official crawler](https://github.com/activitynet/ActivityNet/tree/master/Crawler/Kinetics). Note that this might take a long time.
-Some videos in the ActivityNet dataset might be no longer available on YouTube, so that after video downloading, the downloading scripts update the annotation file to make sure every video in it exists.
 
 ```shell
 bash download_videos.sh
 ```
 
+Since some videos in the ActivityNet dataset might be no longer available on YouTube, official [website](http://activity-net.org/) has made the full dataset available on Google and Baidu drives.
+To accommodate missing data requests, you can fill in this [request form]() provided in official [download page](http://activity-net.org/download.html) to have a 7-day-access to download the videos from the drive folders.
```
Comment (innerlee) [inline tools/data/activitynet/README.md:69]: empty link for request form

File: tools/data/activitynet/download_annotations.sh [hunk 2]
```diff
@@ -7,10 +7,8 @@ fi
 
 cd ${DATA_DIR}
 
-wget https://raw.githubusercontent.com/wzmsltw/BSN-boundary-sensitive-network/master/data/activitynet_annotations/anet_anno_action.json
+wget http://ec2-52-25-205-214.us-west-2.compute.amazonaws.com/files/activity_net.v1-3.min.json
```
Comment (innerlee) [inline tools/data/activitynet/download_annotations.sh:10]: double check http vs https

File: tools/data/activitynet/download_annotations.sh [hunk 3]
```diff
@@ -7,10 +7,8 @@ fi
...
 
-wget https://raw.githubusercontent.com/wzmsltw/BSN-boundary-sensitive-network/master/data/activitynet_annotations/video_info_new.csv
-
-wget https://download.openmmlab.com/mmaction/localization/anet_activity_indexes_val.txt
+wget
```
Comment (innerlee) [inline tools/data/activitynet/download_annotations.sh:12]: ?

File: tools/data/activitynet/download_annotations.sh (hunk 2 above)
Comment (dreamerlin) [reply tools/data/activitynet/download_annotations.sh:10]: https is invaild, http is connectable

Comment (innerlee) [reply tools/data/activitynet/download_annotations.sh:10]: okay

**Comment by dreamerlin** [comment]
Test  with success

**Comment by dreamerlin** [comment]
Kindly Ping @kennymckormick @innerlee

**Comment by innerlee** [comment]
I'm ok with the change as long as it runs successfully.

How about `is_bsn_case` -> `use_bsn_annotation`?

**Comment by kennymckormick** [comment]
I'm OK with that, is there any performance change when testing on the new validation set?

**Review by innerlee — APPROVED** [review]

**Pull request merged**